inference-engine/tests/functional/plugin/gna/pass_tests/remove_permutations_NHWC_to_NCHW_pass.cpp

```diff
@@ -83,15 +83,16 @@ class RemovePermutationsNHWCToNCHWPassTest : public testing::WithParamInterface<
 
             size_t num_out_channels = 12;
             size_t kernel_size = 8;
-            std::vector<size_t> kernal_shape = (inputShape[1] == 1 ? std::vector<size_t>{1, kernel_size} : std::vector<size_t>{kernel_size, 1});
-            auto conv1 = ngraph::builder::makeConvolution(permute1, ngPrc, kernal_shape, { 1, 1 }, { 0, 0 }, { 0, 0 }, { 1, 1 },
+            std::vector<size_t> kernel_shape = (inputShape[1] == 1 ? std::vector<size_t>{1, kernel_size} :
```

Comment:
Can we wrap this line by function since it is duplicated many times in tests?

Reply:
function added